panda/src/express/memoryUsage.cxx

```diff
@@ -520,7 +520,7 @@ MemoryUsage(const MemoryHook &copy) :
 #error Cannot compile MemoryUsage without malloc wrappers!
 #endif
 
-#if (defined(WIN32_VC) || defined(WIN64_VC)) && defined(_DEBUG)
+#ifdef _WIN32
```

Comment:
`_CrtSetAllocHook` is a function in the MSVC runtime and not part of Windows. Perhaps `_MSC_VER` instead?